dt-assets/functions/enqueue-scripts.php

```diff
@@ -265,7 +274,7 @@ function dt_site_scripts() {
             $dependencies[] = 'mapping-drill-down';
         }
 
-        dt_theme_enqueue_script( 'dt-settings', 'dt-assets/js/settings.js', $dependencies, true );
+        dt_theme_enqueue_script( 'dt-settings', 'dt-assets/js/bb', $dependencies, true );
```

Comment:
this looks accidental

Reply:
oops, yes, i must have hit my keyboard at the wrong moment and not noticed 🤦🏻‍♂️

Reply:
resolved with 1a291dd